app/user/rest.py

```diff
@@ -1012,14 +1024,15 @@ def deactivate_user(user_id):
                     continue
```

Comment:
Correct me if I'm wrong - I think this logic means that services that were previously suspended that now have 0 members will not be deactivated. I think we need to handle that case too

Reply:
Yes good catch, let me revise that.